otherlibs/unix/getgroups.c

```diff
@@ -24,19 +25,32 @@
 #include <unistd.h>
 #endif
 #include <limits.h>
```

Comment:
Sorry to be late to the review, but I think `<limits.h>` is no longer needed now that the code does not use `NGROUPS_MAX` anymore.